docs/UserGuide.md

```diff
@@ -60,22 +61,22 @@ Recipes are the building blocks of RIZZipe! Each recipe comprises multiple field
 | **Name**        | `n/`   | **`NAME`** <br> `NAME` can contain these characters: `A-Z a-z 0-9 - \ ' " and space`.                                                                       | Yes         | `Chicken Rice`, <br>`Broccoli Stir-Fry`                                                                                                 |
```

Comment:
## Common Terminologies and Shortcuts

Here are some common terminologies and features that we think will help you better utilise the app:

### What is a `Recipe`?

...

### What is a Recipe `Ingredient` ?

...

### Cool! Can I use my keyboard to navigate around?

Reply:
hm this works. although i think the keyboard shortcuts part isnt necessary, we can just have a one-liner like the current one:
> For experienced users, you may refer to the [Command summary](https://zhtang29.github.io/tp/UserGuide.html#command-summary) and [Keyboard shortcuts summary](https://zhtang29.github.io/tp/UserGuide.html#keyboard-shortcuts-summary) for a summarised table of all the commands and keyboard shortcuts available.